templates/data/prow-periodics-data.yaml

```diff
@@ -60,6 +60,26 @@ templates:
                     - "pubsub_labels"
                     - "disable_testgrid"
                     - extra_refs_test-infra
+              - jobConfig:
+                  name: post-test-infra-markdown-index-autobump
+                  slack_channel: kyma-prow-alerts
+                  branches:
+                    - ^main$
+                  run_if_changed: '.md$'
+                  skip_report: "false"
+                  max_concurrency: "10"
+                  type_postsubmit: "true"
+                  image: eu.gcr.io/kyma-project/test-infra/markdown-index:PR-6386
```

Comment:
Providing image though a globalconfig instead of directly in job would allow autobumper to bump this image

Reply:
fixed